packages/fast-layouts-react/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@microsoft/fast-layouts-react",
+  "version": "1.0.0",
+  "description": "FAST Microsoft app shell written in React",
```

Comment:
@nicholasrice probably want to broaden the description to just layouts with the change